part-2-server/server.js

```diff
@@ -0,0 +1,36 @@
+var http = require('http');
+var fs = require('fs');
+
+var server = http.createServer(function(request, response){
+	var url = request.url;
+	if(url === '/'){
+		fs.readFile('index.html', function(err, data){
+			if(err) throw err;
+			else{
+				response.write(data.toString());
+				response.end();
+			}
+		})
+	} else if(url === '/page-2'){
+		fs.readFile('page-2.html', function(err, data){
+			if(err) throw err;
+			else{
+				response.write(data.toString());
+				response.end();
+			}
+		})
+	} else if (url === '/poem-*')  {
```

Comment:
Good idea...but can't use * in this instance. * will select all in a .gitignore file, but not in an http server